templates/internet-of-things/appstore.html

```diff
@@ -14,147 +14,186 @@
+      <li class="p-matrix__item">
+        <h3 class="p-heading--4">Usage analytics</h3>
+        <p>Track the use of your devices over a dashboard. The Dedicated Snap Store uses the APIs provided by Ubuntu Core to extract usage metrics and present them through a user-friendly interface.</p>
+      </li>
+    </ul>
   </div>
+</section>
 
+<section class="p-strip--light">
   <div class="row">
-    <div class="col-3">
-      <ul class="p-list u-no-margin--bottom">
-        <li class="p-list__item is-ticked">Reliable OTA (over-the-air) updates</li>
-        <li class="p-list__item is-ticked">Secure authentication</li>
-      </ul>
+    <div class="col-6">
+      <h2>Check out the dedicated Snap Store</h2>
     </div>
-
-    <div class="col-3">
-      <ul class="p-list u-no-margin--bottom">
-        <li class="p-list__item is-ticked">CI/CD integration</li>
-        <li class="p-list__item is-ticked">Management dashboard</li>
+    <div class="col-6">
+      <p>The Dedicated Snap Store has a range of features on it's dashboard, including:</p>
+      <ul>
+        <li>Snap inclusion and exclusion</li>
+        <li>Account delegation and role management</li>
+        <li>Release management for your snaps</li>
+        <li>Detailed snap metrics</li>
       </ul>
+      <p>Ready to find out more?</p>
+      <a href="/internet-of-things/contact-us?product=appstore" class="p-button--positive"> Get in touch</a>
     </div>
+  </div>
+</section>
 
-    <div class="col-3">
-      <ul class="p-list u-no-margin--bottom">
-        <li class="p-list__item is-ticked">Usage analytics</li>
-        <li class="p-list__item is-ticked">Multi-tenancy</li>
-      </ul>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Learn more about <br>the Snap Store</h2>
+    </div>
+    <div class="col-6">
+      <h3 class="p-heading--5 u-no-margin--bottom">OTA Updates</h3>
+      <p>
+        <a href="/engage/ota-updates-iot-guide">How to implement over-the-air updates for IoT</a>
+      </p>
+      <h3 class="p-heading--5 u-no-margin--bottom">Enterprise Ready</h3>
+      <p>
+        <a href="/engage/enterprise-snap-management">How to manage Snaps in an Enterprise environment</a>
+      </p>
+      <h3 class="p-heading--5 u-no-margin--bottom">IoT Challenges</h3>
+      <p>
+        <a href="/engage/top-five-iot-challenges">Top 5 IoT challenges and how to solve them</a>
+      </p>
     </div>
   </div>
 </section>
 
-<section class="p-strip">
+<section class="p-strip--light">
   <div class="u-fixed-width">
-    <h2 class="p-muted-heading u-sv3">Read about the companies using app stores for IoT</h2>
-    <div class="p-logo-section has-misaligned-images">
-      <div class="p-logo-section__items">
+    <div class="p-logo-section">
+      <h2 class="p-text--small-caps">Learn about the companies using app stores</h2>
+      <p class="p-text--small">Click each logo to read a case study.</p>
+      <div class="p-logo-section__items u-no-margin--bottom">
```

Comment:
Requested proper documentation on the doc for this copy, not at all clear what the final version is supposed to be